pkgs/development/ocaml-modules/bitstring/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, fetchzip, buildOcaml, time, autoconf, automake }:
+{ stdenv, fetchurl, buildOcaml, time, autoconf, automake }:
 
 buildOcaml rec {
   name = "bitstring";
-  version = "f1673f8"; 
-  src = fetchzip {
-    url = https://storage.googleapis.com/google-code-archive-source/v2/code.google.com/bitstring/source-archive.zip;
-    sha256 = "03343yggwp3y483zj5axaalxlnl698xrjiv3hmd4c2s05g53iwas";
+  version = "2.1.0";
+  src = fetchurl {
+    url = http://github.com/xguerin/bitstring/archive/v2.1.0.tar.gz;
+    sha256 = "0miw4banfpmx4kxrckpqr57b1fcmsqdmspyjx6gqjd4kghm4l7xj";
   };
 
-  patches = [ ./camlp4-git.patch ./meta.patch ./srcdir.patch ];
+  patches = [ ./camlp4-git.patch ];
 
```

Comment:
On darwin, I get the following trace:

```
building
make flags: SHELL=/nix/store/rafidkx85sv66c1q9iplq6zkjhs2136a-bash-4.4-p12/bin/bash
{ echo '/* DO NOT EDIT! GENERATED AUTOMATICALLY! */'; \
  cat /byteswap.in.h; \
} > byteswap.h-t
cat: /byteswap.in.h: No such file or directory
make: *** [Makefile:127: byteswap.h] Error 1
builder for ‘/nix/store/mgjnwki5lpknbi3c3fczm0fr5nmlj79j-ocaml-bitstring-2.1.0.drv’ failed with exit code 2
```

Wild guess: the issue does not show up on Linux since the `byteswap.h` file is found in the libc.

You may remove the patch, if you prefer something like: `makeFlags = "srcdir=.";`.